### ISSUE TITLE
Publications that are both full audio and synchronized audio/text

### ISSUE THREAD

**Issue by HadrienGardeur** [w3c/publ-a11y#399]
Looking at [the technique for displaying metadata for pre-recorded audio in EPUB](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/2.0/techniques/epub-metadata/index.html#pre-recorded-audio) I noticed that it's not currently possible to combine "Audio only" and "Synchronized audio and text".

While it's true that some EPUB publications with Media Overlay only provide pre-recorded audio for some resources, there are a number of publications that also provide synchronized pre-recorded audio and text over the entire publication.

I think that we should re-consider instructions for this technique to check for the case where `accessModeSufficient` contains `auditory` on its own AND `accessibilityFeature` contains `synchronizedAudioText` ([a typo for this value is repeated multiple times throughout the current draft](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/techniques/epub-metadata/#pre-recorded-audio:~:text=accessibilityfeature%3D%22sychronizedaudiotext%22)).

The same technique for ONIX [covers this use case with its third step](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/2.0/techniques/onix-metadata/index.html#instructions-2): 
> ELSE IF all_content_pre_recorded AND synchronised_pre_recorded_audio:
THEN display "Synchronized audio and text".

**Comment by GeorgeKerscher** [comment]
The section 4.4Prerecorded audio
has undergone extensive revision. I would ask @HadrienGardeur 
to review that section and let us know if the concerns have been addressed. If they are please close. If not, let us now what the problems are.


The

**Comment by HadrienGardeur** [comment]
Let's use an example with the following metadata:

- `"accessModeSufficient": [["textual"], ["auditory"]]`
- `"accessibilityFeature": "synchronizedAudioText"`

This could be a novel where the entire text is available as audio with full synchronization using SMIL.

Based on the latest editor's draft (December 23rd), here's what would be displayed:

- **Supports nonvisual reading:** Readable in read aloud or dynamic braille
- **Prerecorded audio:** Audio only

I don't think that's a very accurate representation for these metadata.
It would be better to display something like:

- **Supports nonvisual reading:** Readable in read aloud or dynamic braille
- **Prerecorded audio:** Full content available as audio with synchronized text and audio

IMO the use of "Audio only" is misleading and should only be displayed when `auditory` is the only value available in `accessMode`.

**Comment by GeorgeKerscher** [comment]
I agree!

What it should say as per the updated guidelines is:
• All the content is available as prerecorded audio synchronized with text

If the techniques do not produce this output based on the metadata Hadrien provided, the logic is incorrect.

Also, in the prerecord audio section, it seems the only two options are audio book or audio book synchrinzed with text. 

Should this section be renamed to Audio Book?

Will a small prerecorded audio snip, for example a recording of a poem be in the additional information section? If yes, we do not have an example of this yet.

So I see the two options of complementary audio (the poem reading) would be in Prerecordido or in additional information.

**Comment by clapierre** [comment]
Should be addressed in latest PR #600 at least the part of the logic with both synchronized audio and AMS containing "audio"

**Comment by clapierre** [comment]
This has been addressed. @HadrienGardeur  please reopen if this is still an issue.
